flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/TableSchema.java

```diff
@@ -231,6 +260,15 @@ public String toString() {
 		for (int i = 0; i < fieldNames.length; i++) {
 			sb.append(" |-- ").append(fieldNames[i]).append(": ").append(fieldDataTypes[i]).append('\n');
 		}
+		if (primaryKey.length > 0) {
+			sb.append(" |-- ").append("PRIMARY KEY （").append(String.join(", ", primaryKey)).append("）\n");
```

Comment:
Could you add some more documentation to the `TableSchema` class to explain the semantics of primary key and unique key?

Reply:
Sure. I will add docs later.